invenio_cli/helpers/commands.py

```diff
@@ -357,7 +357,7 @@ def containerize(self, force, install):
             click.secho("Creating files location...", fg="green")
             self.docker_helper.execute_cli_command(
                 project_shortname,
-                "invenio files location --default default-location "
+                "invenio files location create --default default-location"
```

Comment:
```suggestion
                "invenio files location create --default default-location "
```

Reply:
That's what I thought, I reverted back. But it was failing for me (before all the changes on files-rest so it's independent)...

Let's test more in depth after we decide what to do on S3.